src/dataLoaders.ts

```diff
@@ -1,38 +1,33 @@
-import { readFileSync } from 'fs';
-import { CheckType } from './enums';
-import { ModulusWeight } from './interfaces';
+import { readFileSync, writeFileSync } from 'fs';
 
-const loadSubstitutionMap = (): { [key: string]: string } =>
-  readFileSync(`${__dirname}/data/scsubtab.txt`, 'utf8')
-    .split('\r\n')
-    .map((line) => line.split(/\s+/))
-    .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
+const processSubstitutionMap = () =>
+  {
+    const scsubtab = readFileSync(`${__dirname}/data/scsubtab.txt`, 'utf8')
+      .split('\r\n')
+      .map((line) => line.split(/\s+/))
+      .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
+
+    writeFileSync(`${__dirname}/data/scsubtab.json`, JSON.stringify(scsubtab, null, 2));
+    
+  };
 
-const loadModulusWeights = (): ModulusWeight[] => {
-  return readFileSync(`${__dirname}/data/valacdos-v7-90.txt`, 'utf8')
+const processModulusWeights = () => {
+  const valacdos =  readFileSync(`${__dirname}/data/valacdos-v7-90.txt`, 'utf8')
     .split('\r\n')
     .map((line) => {
       const data = line.split(/\s+/);
       return {
         start: parseInt(data[0], 10),
         end: parseInt(data[1], 10),
-        check_type: data[2] as CheckType,
+        check_type: data[2],
         exception: parseInt(data[17], 10) || null,
         weights: data.slice(3, 17).map((weight) => parseInt(weight, 10)),
       };
     });
-};
 
-// we implement a cache here to avoid reading either file multiple times
-const cachedModulusWeights: ModulusWeight[] | undefined = undefined;
-const cachedSubstitutionMap: { [key: string]: string } | undefined = undefined;
-
-export const fetchModulusWeights = () => {
-  if (cachedModulusWeights) return cachedModulusWeights;
-  return loadModulusWeights();
+  writeFileSync(`${__dirname}/data/valacdos.json`, JSON.stringify(valacdos, null, 2));
```

Comment:
We have removed the cache, is there no need for it anymore ? :thinking

Reply:
there's no caching - we write out once and the module itself only works on the converted json files

i think `dataLoaders.ts` need to be renamed to some other name to make it clearer